.github/workflows/semver_checks.yml

```diff
@@ -58,9 +58,7 @@ jobs:
     - name: Setup rust toolchain
       uses: actions-rust-lang/setup-rust-toolchain@v1
     - name: Install semver-checks
-      uses: taiki-e/install-action@v2
-      with:
-        tool: cargo-semver-checks
+      run: cargo install --git https://github.com/Lorak-mmk/cargo-semver-checks.git cargo-semver-checks --branch main
```

Comment:
:wrench: Please add a comment that explains what we are doing and why (temporary solution, fork because ...).